set.mm

```diff
@@ -147746,6 +147746,473 @@ of an open range of nonnegative integers (of length equal to the length of
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Prefixes of a word
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+
+  In ~ https://www.allacronyms.com/prefix/abbreviated , "pfx" is proposed as
+  abbreviation for "prefix".  Regarding the meaning of "prefix", it is
+  different in computer science (automata theory/formal languages) compared
+  with linguistics: in linguistics, a prefix has a meaning (see Wikipedia
+  "Prefix" ~ https://en.wikipedia.org/wiki/Prefix ), whereas in computer
```

Comment:
No need for these five lines, IMO.

Reply:
Which five lines exactly? The section comment consists of 10 lines...

Reply:
The five quoted lines. That is, you can begin at "A prefix is an arbitrary..."

Reply:
OK. I have restructured this text a little bit, because I would like to keep this background information.

Reply:
The whole section header comment is removed now. Some of its contents is moved to the comment of df-pfx.